tox.ini

```diff
@@ -19,8 +19,6 @@ basepython =
 
 deps =
     django-sendfile==0.3.6
```

Comment:
It doesn't look like the changes to this file are part of this PR, could you please revert these.